blog/docs/articles/cross-namespace-event-links-feature.md

```diff
@@ -0,0 +1,123 @@
+
+Over the past three months, I’ve had the incredible opportunity to participate in the LFX Mentorship Program. My specific project was titled CNCF - Knative Eventing: Cross Namespace Event Links. As the term wraps up, I want to take a moment to reflect on my journey and share it with you.
+
+## Feature Description
+
+In a multi-tenant scenario, some use cases might require separating brokers and channels in different namespaces than the services they send events to. While this setup would work if the team consuming the events has access to the namespaces where brokers and channels are in, this is not always the case. To address this issue, a new feature was proposed for triggers and subscriptions (event links) to exist in a different namespace from the broker or channel they reference.​​ The LFX Mentorship Program gave me the chance to bring this novel and highly requested feature into reality. 
+
+For the specific implementation, I used Role-Based Access Control (RBAC) to manage permissions securely. A new RBAC verb, “knsubscribe”, was created to allow users to subscribe to a particular broker/channel. In addition, I used Subject Access Reviews to ensure only authorized users could create and manage these cross-namespace event links, further enhancing security within the Knative Eventing framework. 
+
+Following the new changes, the control plane and data plane were adjusted accordingly to enable the smooth execution of the feature. Unit tests were created to ensure the feature works as expected and an E2E test is currently in progress to confirm event delivery between the resources. All the changes were placed behind a feature flag, allowing users to enable/disable this feature as desired. 
+
+## Using the Cross Namespace Event Links Feature
+
+This feature is planned to come out in the 1.15 release as an alpha feature. Here’s how end users can use cross namespace referencing in Knative Eventing: 
+
+### Enabling the Feature Flag
+
+All the implementation is placed behind a feature flag, `cross-namespace-event-links`. To enable a feature, you can add it to the `config-features` ConfigMap under the `data` spec and set the value of the feature flag to be `enabled`. For example, you can enable the feature by adding the following ConfigMap entry: 
+
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-features
+  namespace: knative-eventing
+  labels:
+    eventing.knative.dev/release: devel
+    knative.dev/config-propagation: original
+    knative.dev/config-category: eventing
+data:
+  cross-namespace-event-links: enabled
+```
+
+### Setting Up Role-Based Access Control (RBAC)
+
+RBAC policies define the permissions required for different resources to interact across namespaces. Here’s how you can set it up: 
+
+**Create a Service Account:** In the namespace where the event source resides, create a service account that will be used to access resources in the target namespace.
```

Comment:
I think here it doesn't necessarily have to be to a service account, it can be to any user. And it may be easier to verify if the users have credentials for a different user in the cluster with less permissions if they make the RoleBinding to that user. Maybe we can clarify this a litte?